zktest/src/main/webapp/test2/config.properties

```diff
@@ -3078,28 +3078,39 @@ B90-ZK-4431.zul=A,E,Multislider
 ##zats##B96-ZK-5126.zul=A,E,Chosenbox,escape,XML,emptyMessage
 ##zats##B96-ZK-5095.zul=A,E,Websocket,Serialization,Cluster
 ##manually##B96-ZK-5149.zul=A,E,dnd,dropupload,scroll,anchor,position
+##zats##B96-ZK-5155.zul=A,E,Listbox,Bandbox,Bandpopup
+##manually##B96-ZK-5170.zul=A,E,Borderlayout,Theme,CSS
+##manually##B96-ZK-5220.zul=A,E,Websocket,tomcat,partial,buffersize
+##zats##B96-ZK-5029.zul=A,E,Listbox,Tree,Grid
+##zats##B96-ZK-5159.zul=A,E,scrollbarWidth
+##manually##B96-ZK-5115.zul=A,E,Websocket,tomcat,thread,WebAppCleanup
+##zats##B96-ZK-5179.zul=A,E,zuti,shadow,apply,variables
 ##zats##B96-ZK-2259.zul=A,E,zuti,tabbox,tab,selected,visible
+##manually##B96-ZK-5253.zul=A,E,Borderlayout,Theme,atlantic,CSS
+##manually##B96-ZK-5240.zul=A,E,Grid,Scrollbar
 ##zats##B96-ZK-5247.zul=A,E,onSize,hflex,grid,rows
 ##zats##B96-ZK-5033.zul=A,E,javascript,endless
 ##zats##B96-ZK-4974.zul=A,E,hflex,grid,column
 ##zats##B96-ZK-5030.zul=A,E,resize,column,listheader,listbox,cssflex,hflex
 ##zats##B96-ZK-5111.zul=A,E,require.js,jserror,SignaturePad,Toast
+##zats##B96-ZK-5223.zul=A,E,zuti,shadow,mergeShadowTree
+##zats##B96-ZK-5176.zul=A,E,Signature,URL,backgroundImage
+##zats##B96-ZK-4982.zul=A,E,Select,mold,Listitem,Listgroup,Optgroup
 ##zats##B96-ZK-5153.zul=A,E,window,invalidate,cssflex,vlayout,flex
 ##zats##B96-ZK-5167.zul=A,E,showBusy,zIndex
 ##zats##B96-ZK-5168.zul=A,E,groupbox,vflex,css,transform,scale,
 ##zats##B96-ZK-5177.zul=A,E,Filedownload,Signature,Canvas,Scale,ZoomOut,DevicePixelRatio
 ##zats##B96-ZK-5181.zul=A,E,Component,Performance,EventQueue
 ##zats##B96-ZK-5148.zul=A,E,datebox,jserror,constraint
+##zats##B96-ZK-5156.zul=A,E,Number,format,BigDecimal,locale,precision,fractionDigits
+##zats##B96-ZK-5180.zul=A,E,Bandpopup,CalendarPop,Datebox,tabindex
 ##zats##B96-ZK-5205.zul=A,E,ClientInfo,DevicePixelRatio
 ##zats##B96-ZK-5203.zul=A,E,Window,invisible,onSize,highlighted
+##manually##B96-ZK-5116.zul=A,E,Multithread,ExValue,NullPointerException
 ##zats##B96-ZK-5208.zul=A,E,AbstractTreeModel,Tree,Performance,CE
 ##zats##B96-ZK-5215.zul=A,E,I18N,EncodedURL
-##manually##B96-ZK-5115.zul=A,E,Websocket,tomcat,thread,WebAppCleanup
-##zats##B96-ZK-5179.zul=A,E,zuti,shadow,apply,variables
-##manually##B96-ZK-5220.zul=A,E,Websocket,tomcat,partial,buffersize
+##zats##B96-ZK-4973.zul=A,E,Tab,Tabs,invalidate,beforeChildReplaced_
 ##zats##B96-ZK-3543.zul=A,E,shadow,selector
-##zats##B96-ZK-5223.zul=A,E,zuti,shadow,mergeShadowTree
-##manually##B96-ZK-5116.zul=A,E,Multithread,ExValue,NullPointerException

```

Comment:
weird removed and added

Reply:
For the release-note and config.properties, I followed the order in 9.6.3, so the diff between every commit with head of master will show the progress.